README.md

```diff
@@ -102,59 +115,66 @@ The development server is split into different categories, which are:
 - general: general discussion channels,
 - development: channels containing development tasks and discussions, also a great place to ask for help,
 - testing: different channels that are used for testing purposes.
+- .ENV: channels used for testing in channel-locked mode.
 
 ### Testing channels
 
-Testing channels are categorised based on the project branch they should be used in conjunction with (more detail in [git flow](#workflow)).
+Testing channels are categorized based on the project branch they should be used in conjunction with (more detail in [git flow](#workflow)).
 
 - release-testing: will only be activated to give a final test to the bot before the code is deployed
 - development-testing: general testing of the code in the _develop_ branch
-- feature-testing-x: branches to test bots in feature branches
-- special-channel-x: channels corresponding to specific channels in the CB server
+- feature-testing-x: channels to test using self-hosted bots running code in feature branches
 
 ## Workflow
 
-To keep the project smooth and running, it is important to have a rigorous development process and a standardised way of doing things.
-
-The following sections contain information about branches, and how they should be treated. If you aren't familiar with how to work with git, please familiarise yourself with its basics, and remember to ask for help from other devs should you need to. Proper git workflow and usage is crucial for the smooth operation of the project.
+To keep the project smooth and running, it is important to have a rigorous development process and a standardized way of doing things.
 
-> **Note:** more in-depth documentation/help available on the [Kyuuto Docs](https://kyuuto.io/kb/yuuto-flow/).
+The following sections contain information about branches, and how they should be treated. If you aren't familiar with how to work with git, please familiarise yourself with its basics, and remember to ask for help from other devs should you need to. Proper git workflow and usage are crucial for the smooth operation of the project.
 
-### Master branch - Kyuuto
+### Master branch - YuuPy
 
-The _master_ branch is the most important branch and the one that contains the running code of Kyuuto (Yuuto). This branch is protected and nobody, not even the maintainers can push code to it directly. The only way code can make its way to the _master_ branch, is when _release_ gets merged with it by a maintainer.
+The _master_ branch is the most important branch and the one that contains the running code of YuuPy (Yuuto). This branch is protected, and nobody, not even the maintainers can push code to it directly. The only way code can make its way to the _master_ branch, is when _release_ gets merged with it by a maintainer.
 
 ### Release branch - BabyShark
```

Comment:
Changing up this weird flow a bit, the current one causes the branches to go out of sync
```suggestion
### Master branch - BabyShark
```